typings/pencil-code-embed-defs.d.ts

```diff
@@ -0,0 +1,88 @@
+// Code - https://pencilcode.net/lib/pencilcodeembed.js
+// Type definitions are only added for PencilCodeEmbed class
+// since it is the only class used in our codebase.
```

Comment:
Move some of it to previous line (81 characters is the limit).

Reply:
Done!